crates/typst/src/eval/func.rs

```diff
@@ -72,9 +72,14 @@ pub use typst_macros::func;
 /// You can define your own function with a [let binding]($scripting/#bindings)
 /// that has a parameter list after the binding's name. The parameter list can
 /// contain positional parameters, named parameters with default values and
-/// [argument sinks]($arguments). The right-hand side of the binding can be a
-/// block or any other expression. It defines the function's return value and
-/// can depend on the parameters.
+/// [argument sinks]($arguments).
+///
+/// The right-hand side of a function binding can be a block or any other
+/// expression. It defines the function's return value and can depend on the
+/// parameters. The `return` keyword can be used inside a block to exit a
+/// function early an return a single value. If no `return` keyword is present,
```

Comment:
```suggestion
/// function early and return a single value. If no `return` keyword is present,
```

Reply:
Also, this is not entirely accurate. Joining also takes place if there is a `return`, but without an explicit value:
```typ
#let f() = {
  [Hello]
  return
  [World]
}

#f() // -> Hello
```
and even
```typ
#let f() = [
  Hello
  #return
  World 
]

#f() // -> Hello
```

Reply:
Oh, I see. So rather I should say:
> Within a function body, the `return` keyword can be used to exit early. If no explicit return value is given, the body evaluates to the result of joining all expressions preceding the `return`. However, if an explicit value is given, only it is returned.

Reply:
I don't think the "However" is necessary, but other than that this is correct and reads well.